Workspace/src/arm_interface/src/arm_motor_commands_node.cpp

```diff
@@ -0,0 +1,110 @@
+#include <ros/ros.h>
+#include <std_msgs/String.h> 
+#include <std_msgs/Int32.h>
+#include <stdbool.h>
+#include <list>
+
+struct msg_from_topics {
+	bool ik_status; 
+	std::list<float> data_points; 
+};
+
+class InitObjects {
+	private:
+	ros::init(argc, argv, "arm_motor_commands");
+	ros::NodeHandle nh;
+	ros::Publisher pub_user_to_GUI =  nh.advertise<std::list<float> >("GUI", 1000); 
+        ros::Publisher pub_user_to_CAN =  nh.advertise<std::list<float> >("CAN", 1000);
+	std_msgs::Int32 freq = 10; 
+
+	public: 
+	static ros::Publisher getPubGUI() {
+		return pub_user_to_GUI; 
```

Comment:
You can't return a non-stick object from a static function. If you want to do things this way, then make the members static. But I would not recommend to even bother with static in this node